<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-H0BK5RM" failures="1" tests="3" name="Test" time="18.612" errors="0" timestamp="2024-12-20T09:30:46 IST">
  <testcase classname="TestCases.RegisterTestCases" name="VerifyRegistrationWithBlankData" time="1.48">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [alert alert-danger alert-dismissible] but found [Warning: You must agree to the Privacy Policy!]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [alert alert-danger alert-dismissible] but found [Warning: You must agree to the Privacy Policy!]
at TestCases.RegisterTestCases.VerifyRegistrationWithBlankData(RegisterTestCases.java:87)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- VerifyRegistrationWithBlankData -->
  <testcase classname="TestCases.RegisterTestCases" name="VerifyRegistrationWithValidData" time="2.419"/>
  <testcase classname="TestCases.LoginTestCases" name="VerifyLoginWithValidData" time="2.94"/>
  <testcase classname="Resources.BaseClass" name="@BeforeSuite setup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setup -->
  <testcase classname="Resources.BaseClass" name="@AfterSuite endreporttest" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite endreporttest -->
</testsuite> <!-- Test -->
